_features/installable.html

```diff
@@ -2,8 +2,17 @@
 title: Installable
 slug: installable
 description: Keep user's most useful app on their home screen without the hassle of an app store.
+usecases:
+- As a user, I want my favourite web applications (GMail, YouTube,
+  Facebook&hellip;) to appear in my homescree as if they were been installed from
+  the marketplace.
```

Comment:
Nits:
* in -> on
* homescree -> homescreen
* they were been -> they had been

Also, "the marketplace" is ambiguous, since you haven't yet referenced a specific marketplace. If you're referring specifically to Google Play, then I would say that. Otherwise, I would say "an app store."